udata_front/theme/gouvfr/datagouv-components/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Export `Placeholder` component, use reuse last_modified [#546](https://github.com/datagouv/udata-front/pull/546)
 - Add missing `dataset` key to `CommunityResource`type [#551](https://github.com/datagouv/udata-front/pull/551)
+- Add beta admin `dataservice` page [#558](https://github.com/datagouv/udata-front/pull/558)
```

Comment:
I think this changelog entry should not be in the datagouv-components changelog

Reply:
oh no ok!